resources/views/livewire/order/order.blade.php

```diff
@@ -223,8 +223,10 @@ class="mx-auto md:flex md:items-center md:justify-between md:space-x-5">
             <div>
                 <h1 class="text-2xl font-bold text-gray-900 dark:text-gray-50">
                     <div class="flex gap-1.5">
-                        <x-heroicons x-cloak x-show="$wire.order.is_locked" variant="solid" name="lock-closed" />
-                        <x-heroicons x-cloak x-show="! $wire.order.is_locked" variant="solid" name="lock-open" />
+                        <div wire:click="toggleLock()" class="cursor-pointer" wire:flux-confirm.icon.warning="{{  __('Unlock Order') }}|{{ __('Unlocking orders can have unexpected side effects.<br><br>Are you Sure?') }}|{{ __('Cancel') }}|{{ __('Continue') }} }}">
```

Comment:
In my understanding of this, the confirm dialog appears every time, even though I may not have the permission do toggle an order.